indexing-service/src/main/java/org/apache/druid/indexing/worker/IntermediaryDataManager.java

```diff
@@ -63,10 +64,10 @@
  * This class manages intermediary segments for data shuffle between native parallel index tasks.
  * In native parallel indexing, phase 1 tasks store segment files in local storage of middleManagers
  * and phase 2 tasks read those files via HTTP.
- *
+ * <p>
```

Comment:
Hmm, these are actually inserted by Intellij's auto indentation based on the code style. It seems legit?

Reply:
Hmm, I think there is maybe some reason why this isn't in the code style - i remember being told to remove these in the past and to manually adjust intellij to not insert them, I'll see if I can find the reason.

Reply:
I think the reason we usually avoid them is that they look silly, and also aren't adding anything useful since Druid javadocs are mostly read in source form anyway. It's not a big deal either way. Maybe one day someone will add a style rule for it and get rid of them all.

Reply:
👍 Removed it.